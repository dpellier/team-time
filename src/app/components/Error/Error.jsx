
import React from 'react';

const Error = () =>
    <div className='error'>
        <span className='error__label'>
            Sorry, seems like something went wrong...
        </span>
    </div>
;

export { Error };
