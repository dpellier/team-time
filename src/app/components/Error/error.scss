
@import '../../style/colors';

.error {
    display: flex;
    position: absolute;
    top: 0;
    right: 0;
    bottom: 0;
    left: 0;
    align-items: center;
    justify-content: center;

    &__label {
        color: $gray;
        font-size: 32px;
    }
}
