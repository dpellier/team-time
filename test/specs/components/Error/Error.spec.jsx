
import React from 'react';
import renderer from 'react-test-renderer';
import { Error } from '../../../../src/app/components/Error/Error';

describe('<Error />', () => {
    it('is correctly rendered', () => {
        const component = renderer.create(
            <Error />
        );
        const tree = component.toJSON();

        expect(tree).toMatchSnapshot();
    });
});
