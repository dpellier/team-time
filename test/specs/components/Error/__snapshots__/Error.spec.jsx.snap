// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> is correctly rendered 1`] = `
<div
  className="error"
>
  <span
    className="error__label"
  >
    Sorry, seems like something went wrong...
  </span>
</div>
`;
